inputs_land.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: index
+title: External Inputs to the Land Model
+prev: diagram.html
+next: land.html
+gcam-version: v5.3 
+---
+
+GCAM's external land inputs include information on land, carbon, other emissions, and the value of unmanaged land in the historical period. These inputs are required for each [global land unit](common_assumptions.html#global-land-units) and [historical year](common_assumptions.html#historical-years).
+
+
+## External Inputs
+
+### Description
+
+| Name | Description | Type | Source | Resolution | Unit |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| Historical Land Use and Land Cover | Land area by region, land type and year. Land cover data is provided beginning in 1700 in order to spin-up the carbon cycle within GCAM. Crop-specific harvested area is provided only for a single year to downscale FAO data to a subnational level. Similarly, the division between irrigated and rainfed land is provided for a single year only. | External Data Set | <a href="https://github.com/JGCRI/moirai">Moirai</a> | Raw data is by country, water basin and land type. | thousand $$km^2$$ |
+| Historical Harvested Area | Harvested land area by country, crop, and year. | External Data Set | FAO | Raw data is by country, crop, and year | Hectares |
+| Terrestrial Carbon Information | Inputs include potential vegetation and soil carbon density (i.e., carbon density if the land grew to equilibrium), and mature age for vegetation carbon. Note that vegetation carbon contents for crops are calculated from crop yields. All other carbon parameters are external inputs. | External Data Set |  Houghton (1999) and King (1997) | Land type and Biome | Various |
+| Soil time scale | Inputs include the number of years for soil carbon changes to occur. Note that this is not the time to equilibrium, which is much longer. | Assumption | | GCAM geopolitical region | Years |
+| Value of Unmanaged Land | GCAM requires profit rates for all land types in the historical period for calibration. Managed land profit is calculated in the <a href="inputs_supply.html">supply model</a>. For unmanaged land, however, the value is input into the model. | External Data Set | <a href="https://github.com/JGCRI/moirai">Moirai</a> | Raw data is by country, water basin, and land use. | million USD |
+| Share Parameters | GCAM requires the user to specify the logit exponents that determine the substitutability between different leafs and nodes in the land model.These parameters were chosen to produce land supply elasticities comparable to those found in the literature. | Assumption |  | Specified by land node | N/A |
```

Comment:
"These parameters were chosen to produce land supply elasticities comparable to those found in the literature, although it should be noted that there is not a transformation between logit exponents and supply elasticities for all land types"?